scripts/modules/MsIndustryLinks/PUBLISHING.md

```diff
@@ -9,6 +9,7 @@ To make your Industry Link available to customers, create and publish an offer i
 - [PowerShell 7.0+](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.3)
```

Comment:
Line 3 of this file will require updating. I cant comment on the line because it hasnt been changed.

Reply:
Updating to?

Reply:
Found it. Updated.